#
# Copyright (c) 2019 Pilz GmbH & Co. KG
#
# This program is free software: you can redistribute it and/or modify
# it under the terms of the GNU Lesser General Public License as published by
# the Free Software Foundation, either version 3 of the License, or
# (at your option) any later version.
#
# This program is distributed in the hope that it will be useful,
# but WITHOUT ANY WARRANTY; without even the implied warranty of
# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
# GNU Lesser General Public License for more details.
#
# You should have received a copy of the GNU Lesser General Public License
# along with this program.  If not, see <http://www.gnu.org/licenses/>.

nodes:
  prbt_joint_1:
    id: 3
    braketest_required: true
  prbt_joint_2:
    id: 4
    braketest_required: true
  prbt_joint_3:
    id: 5
    braketest_required: true
  prbt_joint_4:
    id: 6
    braketest_required: true
  gripper_joint:
    id: 99
  prbt_joint_5:
    id: 7
    braketest_required: true
  prbt_joint_6:
    id: 8
    braketest_required: true
